flink-runtime/src/main/java/org/apache/flink/runtime/deployment/TaskDeploymentDescriptorFactory.java

```diff
@@ -241,6 +271,23 @@ public static TaskDeploymentDescriptorFactory fromExecutionVertex(
             ExecutionVertex executionVertex, int attemptNumber) throws IOException {
         InternalExecutionGraphAccessor internalExecutionGraphAccessor =
                 executionVertex.getExecutionGraphAccessor();
+        final IntermediateDataSetID intermediateDataSetID =
```

Comment:
intermediateDataSetID -> consumedClusterResultId or consumedClusterDataSetId
It can be easier to understanding. because
1. a vertex can produce partitions as well as consume partitions
2. A result contains multiple result partitions. A cluster result contains multiple cluster result partitions (cluster partition in short).